src/iuse_actor.cpp

```diff
@@ -339,11 +339,14 @@ long explosion_iuse::use( player &p, item &it, bool t, const tripoint &pos ) con
         return 0;
     }
     if( it.charges > 0 ) {
-        if( no_deactivate_msg.empty() ) {
-            p.add_msg_if_player( m_warning,
-                                 _( "You've already set the %s's timer you might want to get away from it." ), it.tname().c_str() );
-        } else {
-            p.add_msg_if_player( m_info, _( no_deactivate_msg.c_str() ), it.tname().c_str() );
+        if (p.has_item(it)) {
```

Comment:
Please follow the Code Style of the project, see doc/CODE_STYLE.md

Reply:
my bad, new at this! 

should be fixed I think.